.github/workflows/unitctl.yml

```diff
@@ -186,4 +186,19 @@ jobs:
         with:
```

Comment:
Is it worth flagging releases made with `workflow_dispatch` as `prerelease: true`?

Reply:
Yes.

Reply:
Then let's do that :) Hopefully separate `unitctl/` tags are rare and we'll eventually be able to roll into the normal cadence of Unit releases (and speed those up, too...)

Reply:
Wait, I thought we were basing them off of unit releases?

Also before you go creating _other_ tags, you'd better check with our tooling if that will break anything...

Reply:
I must have been confused by this...

> Then let's do that :) Hopefully separate unitctl/ tags are rare

Reply:
there would have to be a pattern match for unitctl/*** in the workflow file.